recipes/rsync/all/conanfile.py

```diff
@@ -0,0 +1,138 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration, ConanException
+from conan.tools.gnu import Autotools, AutotoolsToolchain, AutotoolsDeps, PkgConfigDeps
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, replace_in_file, rm, rmdir
+from conan.tools.layout import basic_layout
+from conan.tools.build import can_run
+from conan.tools.apple import is_apple_os
+import os
+from io import StringIO
+import re
+
+required_conan_version = ">=1.60.0"
+
+class RsyncConan(ConanFile):
+    name = "rsync"
+    description = "Rsync utility"
+    topics = ("rsync")
```

Comment:
A topic that matches the name is not quite helpful. This is used in the CCI website to aid in search, and names are already taking into account when searching. Some different keywords might work better

Reply:
You are absolutely correct! Fixed!